hybrasyl/Objects/User.cs

```diff
@@ -2142,7 +2159,7 @@ public void AssailAttack(Direction direction, Creature target = null)
             Enqueue(assail.Packet());
             PlaySound(soundId);
             SendAnimation(assail.Packet());
-            PlaySound(soundId);
+            PlaySound(soundId);           

```

Comment:
please do not add unnecessary whitespace

Reply:
Not sure why this whitespace was added. Will be more careful in the future.